include/oneapi/dpl/pstl/utils.h

```diff
@@ -584,10 +584,10 @@ struct __lifetime_keeper : public __lifetime_keeper_base
 // Bitwise type casting, same as C++20 std::bit_cast
 template <typename _Dst, typename _Src>
 ::std::enable_if_t<
-    sizeof(_Dst) == sizeof(_Src) && ::std::is_trivially_copyable_v<_Dst> && ::std::is_trivially_copyable_v<_Src>, _Dst>
+    sizeof(_Dst) == sizeof(_Src) && ::std::is_trivially_copyable_v<_Dst>&& ::std::is_trivially_copyable_v<_Src>, _Dst>
```

Comment:
Let's revert these clang-format inspired changes; these are totally unrelated to the patch, and we recently agreed that it's OK for clang-format checks to fail if the suggested changes are useless or bad for code readability.